src/test/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/service/RecommendationServiceTest.kt

```diff
@@ -230,6 +230,38 @@ internal class RecommendationServiceTest : ServiceTestBase() {
     assertThat(result.crn).isEqualTo(recommendation.get().data.crn)
   }
 
+  @Test
+  fun `get a draft recommendation for CRN from the database`() {
+    val recommendation = RecommendationEntity(id = 1, data = RecommendationModel(crn = crn, lastModifiedBy = "John Smith", lastModifiedDate = "2022-07-19T12:00:00"))
+
+    given(recommendationRepository.findByCrnAndStatus(crn, Status.DRAFT.name))
+      .willReturn(listOf(recommendation))
+
+    val result = recommendationService.getDraftRecommendationForCrn(crn)
+
+    assertThat(result?.recommendationId).isEqualTo(recommendation.id)
+    assertThat(result?.lastModifiedBy).isEqualTo(recommendation.data.lastModifiedBy)
+    assertThat(result?.lastModifiedDate).isEqualTo(recommendation.data.lastModifiedDate)
+  }
+
+  @Test
+  fun `get the latest draft recommendation for CRN when multiple draft recommendations exist in database`() {
+    val recommendation1 = RecommendationEntity(id = 1, data = RecommendationModel(crn = crn, lastModifiedBy = "John Smith", lastModifiedDate = "2022-07-19T23:00:00"))
+    val recommendation2 = RecommendationEntity(id = 2, data = RecommendationModel(crn = crn, lastModifiedBy = "Mary Berry", lastModifiedDate = "2022-08-01T10:00:00"))
+    val recommendation3 = RecommendationEntity(id = 3, data = RecommendationModel(crn = crn, lastModifiedBy = "Mary Berry", lastModifiedDate = "2022-08-01T11:00:00"))
+    val recommendation4 = RecommendationEntity(id = 4, data = RecommendationModel(crn = crn, lastModifiedBy = "Mary Berry", lastModifiedDate = "2022-08-01T09:00:00"))
+    val recommendation5 = RecommendationEntity(id = 5, data = RecommendationModel(crn = crn, lastModifiedBy = "Harry Winks", lastModifiedDate = "2022-07-26T12:00:00"))
```

Comment:
Will the returned date-times in the response have a UTC timezone marker?
eg `2022-07-26T12:00:00.000Z`

Reply:
This field was put in by Bill the other day. Checking the database, I can't see the Z timezone marker but might be worth @bills7281 confirming how this works.

Reply:
This change? https://github.com/ministryofjustice/make-recall-decision-api/pull/233#discussion_r924290672

Reply:
Checking now @jackmaloney @jonnywyatt

Reply:
The date given here is mocked "2022-08-01T10:00:00", so it doesn't come from the repo.

The actual date String sent to the DB is'2022-07-21T09:56:21.034083274'. It is UTC zoned.
But it's format is not '2022-07-26T12:00:00.000Z'.

I can make a seperate PR to address this :) @jonnywyatt @jackmaloney

Reply:
Bill will address this separately